### ISSUE TITLE
How to calculate FPS/FWT?

### ISSUE THREAD

**Issue by xiaoyufenfei** [Eromera/erfnet_pytorch#34]
Dr. Edurado: Hello!
I have been puzzled recently to ask you, so can you tell me?
I am very interested in your real-time semantic work, ERFNet Efficient Residual Factorized ConvNet, but I have not found a description of FPS from your paper, it is related to FWT: FORWARD PASS TIME IN SECONDS, I also run your program , but I don't know if fwt is the Avg time/img:0.042 output during the training of your program, or the forward time per img tested by the script eval_forwardTime.py under the code eval folder, and this time and batchsize have Is it a big relationship? Also, does fwt need to test twice for training and test sets (ie, are they the same)?
I don't understand very well, can you tell me?
Thank you very much for taking time out of your busy schedule!

**Comment by Eromera** [comment]
Forward time in the paper is the time output by eval_forwardTime.py with the model by absorving the batchnorm layers and running with batch_size=1 with the network in eval mode. Training and test sets should be the same, you can select the resolution in the script but normally we use 1024x512.

**Comment by xiaoyufenfei** [comment]
Hi! Thanks for your reply.
